rcl/test/rcl/test_init.cpp

```diff
@@ -271,3 +280,21 @@ TEST_F(CLASSNAME(TestRCLFixture, RMW_IMPLEMENTATION), test_rcl_get_instance_id)
   ret = rcl_context_fini(&context);
   EXPECT_EQ(ret, RCL_RET_OK);
 }
+
+TEST_F(CLASSNAME(TestRCLFixture, RMW_IMPLEMENTATION), test_rcl_init_options_access) {
+  rcl_init_options_t init_options = rcl_get_zero_initialized_init_options();
+  rcl_ret_t ret = rcl_init_options_init(&init_options, rcl_get_default_allocator());
```

Comment:
@Blast545 aren't we missing an `rcl_init_options_fini()` call somewhere in this test?

Reply:
You are right! Added in the last commit